### ISSUE TITLE
iOS Renewed Certification for App Release - expires on 3/6/2025 - revisit 2/2025

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#7751]
**Description** 
As VAHB QA and RElease Team, we need to update the iOS certification before 3/6/2025 so that we can continue to release to the app store after 3/6

**Notes:** 
- <img width="1832" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXZiQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b8dc8d438a94ed0b16c3cac909cd2d330b45a9b9/image.png" alt="image.png" />
- Narin confirmed this reminder needed a ticket, thus this ticket was created 
- If changes are made by 3/6/2025 we wont be able to release to the App Store

**ACs**
-Per Narin, revoke certs and use fastlane match to generate new ones - [steps](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Engineering/DevOps/Automation%20Code%20Docs/Signing%20Keys/Apple/)

**Comment by narin** [comment]
These certs were actually updated back in March by @timwright12 to fix what I believe were some permission issues.  The new expiration date is now 3/6/2025. Updated ticket title and set a new slack reminder in #va-mobile-app

![Screenshot 2024-04-30 at 10 14 52 AM](https://github.com/department-of-veterans-affairs/va-mobile-app/assets/786704/b5f8aa6e-202d-4dc9-8100-1c823ba997c9)

**Comment by jbergman-oddball** [comment]
@timwright12  is this in reference to the cert we are currently trying to update?
cc: @BeccaTupaj

**Comment by timwright12** [comment]
@jbergman-oddball yes, Becca and I have a call tomorrow with @stephenBarrs and @batemapf to try and get it renewed

**Comment by jbergman-oddball** [comment]
@timwright12 Sounds good.  Thank you for confirming.
CC: @ATeal

**Comment by BeccaTupaj** [comment]
Completed 2/27/2025.
